backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -160,7 +159,7 @@ void postConstruct() {
         try {
             aggregationCommand = objectMapper.readValue(aggregationCommandJson, AggregationCommand.class);
         } catch (JsonProcessingException e) {
-            log.log(Level.SEVERE, "Kafka aggregation payload parsing failed", e);
```

Comment:
`Level.SEVERE` causes a restart? or just refactoring here?

Reply:
No this change is not related to the pod restart. I replaced JUL with the JBoss logger :smile: